rest/route/project.go

```diff
@@ -830,12 +830,12 @@ func (h *projectIDGetHandler) Factory() gimlet.RouteHandler {
 func (h *projectIDGetHandler) Parse(ctx context.Context, r *http.Request) error {
 	h.projectName = gimlet.GetVars(r)["project_id"]
 	h.includeRepo = r.URL.Query().Get("includeRepo") == "true"
-	h.includeParserProject = r.URL.Query().Get("includeParserProject") == "true"
+	h.includeProjectConfig = r.URL.Query().Get("includeProjectConfig") == "true"
```

Comment:
We should make sure to update this in the wiki

Reply:
Included in the description!